### PR TITLE
Change selectedCompanyId to display's company

### PR DESCRIPTION
**Pull request by alex-deaconu**
Show warning only when a search filter is active
Highlight sub-company displays

[stage-2]

### CODE REVIEW

**Comment by alex-deaconu** [comment]
@ezequielc please review. @ghislaineguerin FYI. Thanks!

File: partials/displays-list.html [hunk 1]
```diff
@@ -44,10 +54,12 @@ <h1 id="title" translate>displays-app.title</h1>
         </tr>
       </thead>
       <tbody>
-        <tr class="clickable-row" ng-repeat="display in displays.list" ui-sref="display.details({displayId: display.id})">
-          <td class="col-sm-9">{{display.name}}</td>
-          <td class="col-sm-1 text-center"><span class="add-left"><i class="fa {{display | status}} fa-circle"></i></span></td>
-          <td class="col-sm-2 hidden-xs text-right nowrap">{{display.lastActivityDate | date:'d-MMM-yyyy h:mm a'}}</td>
+        <tr class="clickable-row" ng-class="{ 'bg-warning' : selectedCompayId !== display.companyId }" ng-repeat="display in displays.list" ui-sref="display.change({displayId: display.id, companyId: display.companyId})">
+          <td class="col-sm-5 ellipsis display-name"><span class="ellipsis">{{display.name}}</span></td>
+          <td class="nowrap display-address visible-lg visible-md"><span class="text-muted ellipsis">9291 Lake Street Ann Arbor, MI 48103</span></td>
```
Comment (ezequielc) [inline partials/displays-list.html]: @alex-deaconu I think you missed the address here. Cheers

**Comment by alex-deaconu** [comment]
@ezequielc oops, that's embarrassing. Fixed the issue, good catch. Thanks!

**Comment by ezequielc** [comment]
@alex-deaconu No worries. That's why we have peer review :)  
:+1:

**Pull request merged**